lib/mail_room/mailbox_watcher.rb

```diff
@@ -114,6 +114,8 @@ def run
       @running = true
 
       self.idling_thread = Thread.start do
+        process_mailbox
```

Comment:
I don't think this should be here. This would cause problems if the mailbox has a huge backlog of existing messages.

This does have the downside that any mail that comes in while the process isn't running is going to remain unprocessed. If it's needed, I'm happy to make a new issue of this, and try to add it as a feature.